parlai/core/dict.py

```diff
@@ -116,7 +116,7 @@ def add_cmdline_args(argparser):
             dictionary.add_argument(
                 '--dict-maxtokens', default=DictionaryAgent.default_maxtokens,
                 type=int,
-                help='max number of tokens to include in sorted dict')
+                help='max number of tokens to include in dictionary or bpe codecs.')
```

Comment:
should we make a similar note for minfreq?

Reply:
done